show/syslog.py

```diff
@@ -51,3 +62,69 @@ def syslog(db):
         body.append(row)
 
     click.echo(format(header, body))
+
+@syslog.command(
+    name='rate-limit-host'
+)
+@clicommon.pass_db
+def rate_limit_host(db):
+    """ Show syslog rate limit configuration for host """
+
+    header = [
+        "INTERVAL",
+        "BURST",
+    ]
+    body = []
+    syslog_configs = db.cfgdb.get_table(SYSLOG_CONFIG_TABLE)
+    if SYSLOG_CONFIG_GLOBAL_KEY in syslog_configs:
+        entry = syslog_configs[SYSLOG_CONFIG_GLOBAL_KEY]
+        body.append([entry.get(SYSLOG_RATE_LIMIT_INTERVAL, 'N/A'),
+                    entry.get(SYSLOG_RATE_LIMIT_BURST, 'N/A')])
+
+    click.echo(format(header, body))
+
+
+@syslog.command(
+    name='rate-limit-container'
+)
+@click.argument('service_name', metavar='<service_name>', required=False)
+@clicommon.pass_db
+def rate_limit_container(db, service_name):
+    """ Show syslog rate limit configuration for containers """
+
+    header = [
+        "SERVICE",
+        "INTERVAL",
+        "BURST",
+    ]
+    body = []
+    features = db.cfgdb.get_table(FEATURE_TABLE)
+    syslog_configs = db.cfgdb.get_table(SYSLOG_CONFIG_FEATURE_TABLE)
+
+    if service_name:
```

Comment:
@Junchao-Mellanox suggest to reuse feature name validator

Reply:
where is it?

Reply:
@Junchao-Mellanox you need to write it first. Please check the existing code

Reply:
@Junchao-Mellanox better to have share lib